tests/SharedPluginsAndCodeactivites/Plugin.cs

```diff
@@ -297,284 +314,4 @@ yield return
         #endregion
 
     }
-
```

Comment:
I get why you want to split the Plugin.cs base class up, but if we do that it would be a breaking change that I don't believe is necessary. This base class is what is shipped with DAXIF and used on all of our customers at the moment, so I would like to avoid any breaking changes to it

Reply:
There was a reason for it, but I can't remember what it was :-)  I'll take another look.

Reply:
I've restored it back to a single class.  The issue it was done to prevent was the "complex code" warning from code factor, which has unsurprisingly returned...  I'll let you worry about that one :-)